### PR TITLE
DDCE-5603 Trusts- help with proof of registration.

### PR DESCRIPTION
**Pull request by harish-korada1**

### CODE REVIEW

**Review by PaulC-TDS — DISMISSED** [review]

File: test/controllers/ObligedEntityPdfControllerSpec.scala [hunk 1]
```diff
@@ -105,6 +105,10 @@ class ObligedEntityPdfControllerSpec extends SpecBase {
         val result = route(application, request).value
 
         status(result) mustEqual INTERNAL_SERVER_ERROR
+        // added for testing the DDCE-5603 for custom error template
```
Comment (matthew-gibson-18) [inline test/controllers/ObligedEntityPdfControllerSpec.scala:108]: I can see most views are tested using this sort of format e.g. UnauthorisedControllerSpec.scala: 
      contentAsString(result) mustEqual
        view()(request, messages).toString

File: app/views/CustomErrorTemplate.scala.html
[the remark this answers is not in the document]
Comment (harish-korada1) [reply app/views/CustomErrorTemplate.scala.html:1]: its the error page

File: test/controllers/ObligedEntityPdfControllerSpec.scala (hunk 1 above)
Comment (harish-korada1) [reply test/controllers/ObligedEntityPdfControllerSpec.scala:108]: yeah i check for the contains on the page, so I've used this lines 

contentAsString(result) contains

File: conf/messages [hunk 2]
```diff
@@ -1274,3 +1274,15 @@ individualConfirmation.warning = If you try to make changes using the trust’s
 individualConfirmation.paragraph9 = If any of the trust details change before you get the UTR, keep a record of the changes and make them online once you have the UTR.
 
 leadTrustee.default = the lead trustee
+
+########## DDCE-5603 Error pages ##########
+customError.heading = There is a problem
+customError.paragraph1 = We need to speak to you to help you to get the proof of registration for the trust.
+customError.subheading = Contact the trusts helpline
+customError.paragraph2 = Telephone:
+customError.paragraph2.telePhne = 0300 123 1072
```
[the remark this answers is not in the document]
Comment (harish-korada1) [reply conf/messages]: updated to telephone

**Review by matthew-gibson-18 — APPROVED** [review]

**Review by matthew-gibson-18 — APPROVED** [review]

**Review by matthew-gibson-18 — APPROVED** [review]
looks good

**Pull request merged**